src/recommendations/RecommendationsPage.jsx

```diff
@@ -21,18 +22,36 @@ import RecommendationsLargeLayout from './RecommendationsPageLayouts/LargeLayout
 import RecommendationsSmallLayout from './RecommendationsPageLayouts/SmallLayout';
 import { LINK_TIMEOUT, trackRecommendationsViewed, trackSkipButtonClicked } from './track';
 import { DEFAULT_REDIRECT_URL } from '../data/constants';
+import { getAllPossibleQueryParams } from '../data/utils';
 
 const RecommendationsPage = () => {
   const { formatMessage } = useIntl();
+  const DASHBOARD_URL = getConfig().LMS_BASE_URL.concat(DEFAULT_REDIRECT_URL);
   const isExtraSmall = useMediaQuery({ maxWidth: breakpoints.extraSmall.maxWidth - 1 });
   const location = useLocation();
+  const queryParams = getAllPossibleQueryParams();
+  // flag to show recommendations for onboarding component experience
+  const showRecommendations = !!queryParams?.levelOfEducation
+    || (!!queryParams?.finalRedirectUrl && !!queryParams?.country);
```

Comment:
this would show recommendations even if there is only level of education in query params. I think we should have some other checks as well. maybe we can introduce a new query param? `referrer=onboarding-component`?

Reply:
The condation is with `OR` operator with `levelOfEducation` and (finalRedirectUrl && country), so if there is only level of education in query params the recommendations will be shown.